### DISCUSSION TITLE
how should i profile a pod in agent.

### DISCUSSION THREAD

**Discussion by zdyj3170101136** [bloomberg/memray#528] [General]
i want to deploy a daemonSet in k8s.

which will run memray for each python process of k8s to get why memory is so high.

but as the document said, the memray should build with different version of python version.

there may be tens of python version, like 3.7.1, 3.7.2 and so on in k8s machine.

so how should i fix this problem?

**Comment by sarahmonod** [comment]
I'm pretty sure that by "should build with different versions of python", what was meant is "different *minor* versions of python", so memray built with python `3.7.2` should work with python `3.7.1` and `3.7.3` for example.
